crm_claim_rma/models/account_invoice_line.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
```

Comment:
Hi @pedrobaeza, 

As [File Naming](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming) and [Differences With Odoo Guidelines](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#differences-with-odoo-guidelines) sections in Contributing guidelines shows, any inherited model is a separated file.